### PR TITLE
[github]: add new Frontend_Report / Issue_Report & optimize config.yml

### PR DESCRIPTION
**Pull request by MickLesk**
## Description

1. Add new Frontend_Report for Website
2. Optimize Issue Report
3. Optimize Issue Overview

Old: 
image

New: 
image

## Type of change
Please check the relevant option(s):

- [ ] Bug fix (non-breaking change that resolves an issue)
- [x] New feature (non-breaking change that adds functionality)
- [x] Breaking change (a fix or feature that would cause existing functionality to change unexpectedly)
- [ ] New script (a fully functional and thoroughly tested script or set of scripts.)

## Prerequisites
The following efforts must be made for the PR to be considered. Please check when completed:
- [x] Self-review performed (I have reviewed my code, ensuring it follows established patterns and conventions)
- [x] Testing performed (I have tested my changes, ensuring everything works as expected)
- [x] Documentation updated (I have updated any relevant documentation)

### CODE REVIEW

**Comment by remz1337** [comment]
Not sure about the documentation one. I checked the link the PR and it redirects to frontend issue?

**Comment by MickLesk** [comment]
> Not sure about the documentation one. I checked the link the PR and it redirects to frontend issue?

This is currently intended because the general documentation for LXCs/VMs and other scripts can be found on the website. (CPU/RAM/login etc )

**Review by bramsuurdje — DISMISSED** [review]

**Review by havardthom — COMMENTED** [review]
Have some comments, but eating dinner atm, will add after

**Comment by MickLesk** [comment]
> Have some comments, but eating dinner atm, will add after

Take your time, I'm out for today :-) If then I'll take a look at it tomorrow.

File: .github/ISSUE_TEMPLATE/bug_report.yml
Comment (havardthom) [inline .github/ISSUE_TEMPLATE/bug_report.yml:2]:
```suggestion
description: Report a specific issue with a script. For other inquiries, please use the Discussions section.
```

File: .github/ISSUE_TEMPLATE/bug_report.yml
Comment (havardthom) [inline .github/ISSUE_TEMPLATE/bug_report.yml:3]:
```suggestion
```
Everything is an issue so this doesn't add any value

File: .github/ISSUE_TEMPLATE/bug_report.yml
Comment (havardthom) [inline .github/ISSUE_TEMPLATE/bug_report.yml:4]:
```suggestion
```
We shouldn't overlabel things. No labels is a good indication no one has looked at the issue yet.

File: .github/ISSUE_TEMPLATE/bug_report.yml
Comment (havardthom) [inline .github/ISSUE_TEMPLATE/bug_report.yml:31]: Why change this to a multi-select?

File: .github/ISSUE_TEMPLATE/bug_report.yml
Comment (havardthom) [inline .github/ISSUE_TEMPLATE/bug_report.yml:32]: Move below attributes

File: .github/ISSUE_TEMPLATE/bug_report.yml
Comment (havardthom) [inline .github/ISSUE_TEMPLATE/bug_report.yml:63]: Are the quotes included in the placeholder? If they are, I don't think a placeholder should have quotes

File: .github/ISSUE_TEMPLATE/frontend_report.yml
Comment (havardthom) [inline .github/ISSUE_TEMPLATE/frontend_report.yml:1]:
```suggestion
name: "🌐 Frontend Issue Report"
```

Comment (havardthom) [reply .github/ISSUE_TEMPLATE/frontend_report.yml:1]: Should be consistent with other titles

File: .github/ISSUE_TEMPLATE/config.yml
Comment (havardthom) [inline .github/ISSUE_TEMPLATE/config.yml:5]: This sentence is confusing and contradicting. Do we need a separate Documentation Issue? Wouldn't the previous `website issue` cover both frontend and documentation. Not sure users will bother to separate these issues when it's all on the website.

File: .github/ISSUE_TEMPLATE/config.yml
Comment (havardthom) [inline .github/ISSUE_TEMPLATE/config.yml:8]:
```suggestion
      about: For suggestions or questions, please use the Discussions section.
```
consistent with descriptions in template

File: .github/ISSUE_TEMPLATE/config.yml
Comment (havardthom) [inline .github/ISSUE_TEMPLATE/config.yml:11]:
```suggestion
      about: For feature/script requests, please use the Discussions section.
```

File: .github/ISSUE_TEMPLATE/config.yml
Comment (havardthom) [inline .github/ISSUE_TEMPLATE/config.yml:14]:
```suggestion
      about: Join our Discord server to chat with other users in the Proxmox Helper Scripts community.
```
consistent with README.md

File: .github/ISSUE_TEMPLATE/frontend_report.yml
Comment (havardthom) [inline .github/ISSUE_TEMPLATE/frontend_report.yml:2]:
```suggestion
description: Report an issue specifically related to the frontend. For other inquiries, please use the Discussions section.
```
Generally for this PR we should decide between `frontend` or `website`. Personally I find `website` more user friendly since not everyone know what a frontend is

File: .github/ISSUE_TEMPLATE/frontend_report.yml
Comment (havardthom) [inline .github/ISSUE_TEMPLATE/frontend_report.yml:4]:
```suggestion
labels: "website"
```
if we use the website label we don't need the verbose title (which user also can remove in edit)

File: .github/ISSUE_TEMPLATE/frontend_report.yml
Comment (havardthom) [inline .github/ISSUE_TEMPLATE/frontend_report.yml:3]:
```suggestion
```
not needed when we have website label

File: .github/ISSUE_TEMPLATE/frontend_report.yml
Comment (havardthom) [inline .github/ISSUE_TEMPLATE/frontend_report.yml:12]:
```suggestion
        - If the problem is related to a bug in the frontend, kindly check for browser compatibility and ensure the issue occurs in multiple browsers before submitting.
```
I don't think anyone is gonna do this.

File: .github/ISSUE_TEMPLATE/frontend_report.yml
Comment (havardthom) [inline .github/ISSUE_TEMPLATE/frontend_report.yml:28]: Why restrict the user like this? It could also be something else

**Review by havardthom — COMMENTED** [review]
Quite a few comments here, honestly I just think I like the old templates better. This adds a lot and feels a little bloated to me. In my eyes we have two things which are user-facing: the scripts and a website. Having one template for each feels simpler and more consistent for me.

**Comment by remz1337** [comment]
Really great review @havardthom, I agree with everything

**Comment by MickLesk** [comment]
I absolutely do not agree with every comment. Some comments are also in the existing template from tteck. Somehow it seems like I'm being portrayed as more stupid ^^ 

About the wording, okay but you don't have to optimize every single sentence. It's already hard "educating", like a teacher. But as far as I'm concerned, I can look through it tomorrow.

**Comment by remz1337** [comment]
@MickLesk please don't take this personally. I think you have been doing an excellent job as maintainer so far, don't downplay yourself.

Me agreeing with his review doesn't mean I'm right either ;) I'm just providing my thoughts. This project now being community driven means we may need more time to come to an agreement. Would be great to hear from other contributors too (@community-scripts/contributor)

**Comment by havardthom** [comment]
@MickLesk I'm sorry if I come off as harsh, I'm just voicing my opinions and it is fine to disregard them if people disagree. I do a lot of nitpicking but I feel it is important to keep the consistency and simplicity tteck lived by. 
I absolutely don't think or want to portray you as stupid, I think you do great work for the community and I really appreciate all of it, as I'm sure the rest of the team also does.

**Comment by Mellowlynx** [comment]
As it's getting late, I will be happy to also check and check the feedback sometime tomorrow.

File: .github/ISSUE_TEMPLATE/frontend_report.yml
Comment (MickLesk) [reply .github/ISSUE_TEMPLATE/frontend_report.yml:27]: For Example? 😄 

i can add "Other", but somehow I think the 3 options are already good. But @BramSuurdje can also say something about that

My intention behind it:
I think free text would create a lot of “garbage” here, or it would be misused to directly insert large error parts as text.

File: .github/ISSUE_TEMPLATE/frontend_report.yml
Comment (MickLesk) [reply .github/ISSUE_TEMPLATE/frontend_report.yml:11]: Yes, I am very aware of that, but hardly nobody will do it, except really technical afine who work very fine.

I based this on what tteck did (first run all scripts in verbose and default settings), probably only 5-20% do that :smile:

Comment (Mellowlynx) [reply .github/ISSUE_TEMPLATE/frontend_report.yml:11]: Still good to have, and be able to point to.
Especially if we can't reproduce the problem.

File: .github/ISSUE_TEMPLATE/bug_report.yml
Comment (MickLesk) [reply .github/ISSUE_TEMPLATE/bug_report.yml:29]: looks better and its possible to check both if both try

File: .github/ISSUE_TEMPLATE/bug_report.yml
Comment (MickLesk) [reply .github/ISSUE_TEMPLATE/bug_report.yml:61]: works fine in yaml

image

**Comment by MickLesk** [comment]
So ive updated smart little things. 
I think it looks good, is clearly laid out and contains the most important functions.
The only thing that could be made smarter is the text under the title of the issue report.

**Feedback on this is welcome!** 

You can test it here: 
https://github.com/MickLesk/Proxmox_DEV/issues/new/choose

@BramSuurdje 
Especially for you, you will “work” the most with it. Is the front-end report good, or is it missing something, or is it too much?

**Review by bramsuurdje — APPROVED** [review]

**Review by havardthom — APPROVED** [review]
Looks good, thank you for improving

**Pull request merged**
